### PR TITLE
change text, add account address in reputation connection warning

### PR DESCRIPTION
**Pull request by dkent600**

### CODE REVIEW

File: src/resources/dialogs/connectToNet/connectToNet.html [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+          src="../../../../static/dutchx-blue.svg"
+          class="logo d-inline-block align-top"
+          alt="DutchX"
+        >
+        Welcome to the DutchX Initializer</div>
+      <div
+        else
+        class="d-inline"
+      >
+        <div class="d-inline-block warning-icon">⚠️</div>Important Notice: Reputation is Not Transferable
```
[the remark this answers is not in the document]
Comment (dkent600) [reply src/resources/dialogs/connectToNet/connectToNet.html]: It was requested in the spec.  The message is meant to be a warning.

[the remark this answers is not in the document]
Comment (orenyodfat) [reply src/resources/dialogs/connectToNet/connectToNet.html]: and GitHub show it as a warning as well ?

[the remark this answers is not in the document]
Comment (dkent600) [reply src/resources/dialogs/connectToNet/connectToNet.html]: https://www.fileformat.info/info/unicode/char/26a0/index.htm

[the remark this answers is not in the document]
Comment (orenyodfat) [reply src/resources/dialogs/connectToNet/connectToNet.html]: 👍

**Pull request merged**